documentation/modules/ROOT/pages/transformations/outbox-event-router.adoc

```diff
@@ -148,17 +148,16 @@ To obtain this value from a different outbox table column, set the {link-prefix}
  +
 To obtain the event key from a different outbox table column, set the {link-prefix}:{link-outbox-event-router}#outbox-event-router-property-table-field-event-key[`table.field.event.key` SMT option] in the connector configuration.
 
-|`type`
-|A user-defined value that helps categorize or organize events.
-
-a|`payload`
-|The representation of the event itself. The default structure is JSON. The content in this field becomes one of these:
-
-* Part of the outbox message `payload`.
-* If other metadata, including `eventType` is delivered as headers, the payload becomes the message itself without encapsulation in an envelope.
+|`payload`
+|The representation of the event itself. The default structure is JSON. By default, the content in this field becomes the sole content of the Kafka message value; if further fields are configured to be included in the outbox event, the Kafka message value will contain an envelope structure containing any additional field(s) and the payload in a field `payload`.
```

Comment:
@roldanbob, are you happy with that? If not, let me know. The reason for removing `eventType` is that this isn't any specifically supported field, it's just an example for a potential "additional field".

Reply:
How about:

> A representation of the outbox change event. The default structure is JSON. By default, the `payload` value includes the entire content of the Kafka message. However, if the outbox event is configured to include additional fields, the
> additional fields are encapsulated within the Kafka envelope, and are represented separately from the `payload` field. For more information, see xref:emitting-messages-with-additional-fields[Emitting messages with additional fields].

Not sure whether I've interpreted all of this correctly. I was looking at the 1.6 and 1.7 docs, and I see the following sentence: _If other metadata, including `eventType` is delivered as headers, the payload becomes the message itself without encapsulation in an envelope._ So in that case,  is the payload also treated as just another "additional field"? Not sure how that changes the way that we should describe the fields in the [basic outbox table](https://debezium.io/documentation/reference/1.6/transformations/outbox-event-router.html#basic-outbox-table).

Perhaps this is a more accurate last sentence (before the cross reference).

>However, if the outbox event is configured to include additional fields, the payload and the additional fields are encapsulated within the Kafka envelope, and each field is represented separately.

Reply:
> the payload value includes the entire content of the Kafka message

It's more the other way around. The Kafka message value is solely comprised of the payload in that case. But we're getting there:

> A representation of the outbox change event. The default structure is JSON. By default, the Kafka message value is solely comprised of the `payload` value. However, if the outbox event is configured to include additional fields, the Kafka message value will contain an envelope encapsulating both payload and the additional fields, and each field is represented separately. For more information, see xref:emitting-messages-with-additional-fields[Emitting messages with additional fields].